### ISSUE TITLE
Import + set tags not working in 4.3 Beta and 4.3.1

### ISSUE THREAD

**Issue by jos0405** [mautic/mautic#11194]
### Mautic Version

4.2.x series

### PHP version

php 7.4

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When I try to imoprt a batch of contacts and choose a tag to be applied to all of them - the tag is not applied altough the import works. Just the tag is missing.

### How can we reproduce this issue?

Step 1: create a list of contacts in csv
Step 2: import and set a "tag" for the import
Step 3: your tag is on none of the contacts.

### Relevant log output

```shell
No log output, no visible errors
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by jos0405** [comment]
Cache cleared after upgrade, assets regenerated.

**Comment by poweriguana** [comment]
With the same issue. Waiting for solution

**Comment by justinroffmarsh** [comment]
Also seeing this issue.

**Comment by haneefputtur** [comment]
Same problem i am also facing

**Comment by mceo-jp** [comment]
Same issue , Any solution ?

**Comment by mundey** [comment]
observed same issue, tried to import from batch/browser, single tag, multiple tags. not working. stuck. no errors shown.

understood the problem but not familiar with code to find the fix.

so, import db table in properties column keep track of this info. however, tags always comes empty array

```json
{"fields":{"email":"email","firstname":"firstname","lastname":"lastname"},"defaults":{"owner":null,"list":null,"tags":[],"skip_if_exists":false},"headers":["firstname","lastname","email"],"parser":{"delimiter":",","enclosure":"\"","escape":"\\","batchlimit":100}}
```
someone who has worked on this project should be able to fix it, i am still trying to understand code flow.

**Comment by Dylan-86** [comment]
Same issue here, it's driving me crazy...

**Comment by haneefputtur** [comment]
> Same problem i am also facing

I finally ended up in using 4.2.x version.

**Comment by mundey** [comment]
tldr: still the same issue
i went through the process of setting up dev environment, so profiler can give some clue, the value sent in the post (lead_field_import) shows there are values for tag id present but it does not get stored in the table (i mentioned in previous comment)
```json
lead_field_import = [
  "owner" => "1"
  "list" => ""
  "tags" => [
    "2"
    "1"
  ]
  "skip_if_exists" => "0"
  "email" => "email"
  "firstname" => "firstname"
  "lastname" => "lastname"
  "_token" => "JK_J-Bq_N5Sfv8GupDumhnIWuvVqcaO1QzEGDTIfIYU"
  "buttons" => [
    "save" => ""
  ]
]
```

**Comment by RCheesley** [comment]
Thanks folks!

We had a few PRs in the 4.3 release which touched the import functionality:

https://github.com/mautic/mautic/pull/11060
https://github.com/mautic/mautic/pull/10884
https://github.com/mautic/mautic/pull/11011

I am guessing one of those is the culprit for this regression so hopefully we can snag the bug and get it fixed in the next release. 

@mundey would you be up for creating a PR to fix this issue by any chance? Docs are here: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests - you should use the 4.3 branch.

cc @escopecz @mollux for visibility.

**Comment by escopecz** [comment]
I can see some import tag fixes in our company Mautic fork. I'll push them and see if it will fix it. Stay tuned.

**Comment by escopecz** [comment]
Please test https://github.com/mautic/mautic/pull/11220. We need 2 approvals to get it merged and release with 4.3.2

**Comment by poweriguana** [comment]
@escopecz Tested and worked. Not any problems regards on this found.

**Comment by mundey** [comment]
> Thanks folks!
> 
> We had a few PRs in the 4.3 release which touched the import functionality:
> 
> #11060 #10884 #11011
> 
> I am guessing one of those is the culprit for this regression so hopefully we can snag the bug and get it fixed in the next release.
> 
> @mundey would you be up for creating a PR to fix this issue by any chance? Docs are here: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests - you should use the 4.3 branch.
> 
> cc @escopecz @mollux for visibility.

Really sorry @RCheesley i got busy and could not take a look at the PR. will update to new version and test
